src/backend/Level0/CHIPBackendLevel0.cc

```diff
@@ -814,6 +792,19 @@ hipError_t CHIPKernelLevel0::getAttributes(hipFuncAttributes *Attr) {
 // CHIPQueueLevelZero
 // ***********************************************************************
 
+std::vector<ze_event_handle_t> CHIPQueueLevel0::getEventListHandles(
```

Comment:
FWIW, this function could be a free function instead of a member function of CHIPQueueLevel0.

Reply:
I prefer it being a method